src/base/bittorrent/session.cpp

```diff
@@ -4305,7 +4305,7 @@ namespace
 
         torrentData.savePath = Profile::instance().fromPortablePath(
             Utils::Fs::fromNativePath(QString::fromStdString(fast.dict_find_string_value("qBt-savePath"))));
-        torrentData.ratioLimit = QString::fromStdString(fast.dict_find_string_value("qBt-ratioLimit")).toDouble();
+        torrentData.ratioLimit = std::stod(fast.dict_find_string_value("qBt-ratioLimit"));
```

Comment:
Please add header `#include <string>`

Reply:
But it works?

Reply:
Never mind, there must be a reason if you say so.

Reply:
It works since this header is included indirectly by some other header.
It should be included in torrenthandle.cpp too, because of to_string() function.

Reply:
Will do.

Reply:
Do you realise that certain changes of app or environment locale make these values unreadable? This is certainly not the thing expected from a binary file. We need to either specify locale expliciterly here or avoid converstion between string and numbers.

Reply:
>We need to either specify locale expliciterly here or avoid converstion between string and numbers.

I guess this means going all the way back to `QString`...